frontend/server/src/Controllers/Submission.php

```diff
@@ -128,6 +128,56 @@ private static function createFeedback(
         }
     }
 
+    public static function tryCreateFeedback(
```

Comment:
El tercer parámetro al ser un array, tendrías que especificar mas a detalle que tipo de array recibirá:
```suggestion
    /**
     * @param array{assignment_alias: string, author_id: int|null, course_alias: string, course_id: int, problem_alias: string} $courseSubmissionInfo
     */
     public static function tryCreateFeedback(

```